README.md

```diff
@@ -163,6 +163,104 @@ It is also possible to setup a customer specific build job on request where one
+	libsdl1.2-dev && rm -rf /var/lib/apt/lists/*
+
+	ENV USER_HOME /home/${user}
+
+	RUN groupadd -g ${gid} ${group} \
+	    && useradd -d "$USER_HOME" -u ${uid} -g ${gid} -m -s /bin/bash ${user}
+	# Edit username to your username
+	RUN adduser username sudo
+	# Edit username to your username
+	RUN echo "username ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
+	# Switch to user
+	USER ${user}
+	WORKDIR ${USER_HOME}
+
+	# Creates project folder. Returns no error if folder aldready exists
+	RUN mkdir -p project
+
+	# Edit "your@mail.com"
+	RUN git config --global user.email "your@mail.com"
+	# Edit "Your Name"
+	RUN git config --global user.name "Your Name"
+	RUN git config --global push.default simple
+
+	# END COPY
+
+#### Run Docker image
+	# Edit user id 1000 with your user id. 
+	# Edit username with your username (both in ssh- and project path).
+	# Edit path to the project /home/username/project if needed.
+	# The project folder is the link between your environment and the Docker environment.
+	# Everything you put in the project folder from the Docker will be accessible outside the Docker.
+	# To make --privileged to work, it assumes that you have a SSH key associated with your github account.
+	# If SSH key is not set up correctly, you will be asked for username and password during the git clone process.
+	sudo docker run -it -u 1000 --privileged -v ~/.ssh:/home/username/.ssh -v ~/project:/home/username/project hostmobility/mx4
+
+
+### Example to setup environment for MX-4 CT, branch BSP-v1.5.x
+
+#### Git clone from repository
+	# If SSH key is associated correctly with your account on github, run follow commands:
+	git clone git@github.com:hostmobility/mx4 -b mx4-bsp-v1.5.x
+	git clone git@github.com:hostmobility/mx4-pic -b mx4-bsp-v1.5.x mx4/pic
+	# If not correctly associated, run follow commands:
+	git clone https://github.com/hostmobility/mx4/ -b mx4-bsp-v1.5.x
+	git clone https://github.com/hostmobility/mx4-pic -b mx4-bsp-v1.5.x mx4/pic
+
+#### Build CT-BSP-v1.5.x
+	cd mx4/t20
+	# Remember to edit username to your own username and project if you
+	# named your project folder to something else.
+	# edit $(nproc) to number of cores you want to use while buildning.
+	./make_system.sh -t ct -r /home/username/project/rootfs/CT -d /home/username/project/yocto-1.5.x -g -k -u -j $(nproc) -m -T 512
+
+### Example to setup environment for MX-4 V61
+
+	# All products use mx4 and mx4-pic repository
+	git clone git@github.com:hostmobility/mx4
+	git clone git@github.com:hostmobility/mx4-pic mx4/pic
+
+
+	# Below are only required for products who have Linux and U-boot
+	# outside of the mx4 repository
+	git clone git@github.com:hostmobility/linux-toradex.git -b hm_vf_4.4 mx4/t20/linux_vf
+	git clone git@github.com:hostmobility/u-boot-toradex.git -b 2015.04-hm mx4/t20/u-boot_vf
+
+### Example to build Board Support Package for MX-4 V61
+
+	cd mx4/t20
+	./make_system.sh -t v61 -r ~/rootfs/v61 -u -k -j 8 -g
+
+
```

Comment:
Ta bort raden "work in progress".

Tipsa gärna om att när man vill arbeta i olika brancher så bör man ändra sökvägen i -d argumentet. Arbetar man med flera produkter så bör man ändra sökvägen i -r argumentet.